data/validation/password.go

```diff
@@ -0,0 +1,18 @@
+// Package validation provides validation for data inside widgets
+package validation // import "fyne.io/x/fyne/data/validation"
+
+import (
+	"fyne.io/fyne/v2"
+	gpv "github.com/wagslane/go-password-validator"
```

Comment:
I think  this is missing a newline between the import groups.
Maybe we have not got the full suite of checks running on this repo yet?

Reply:
The workflow is a few versions behind, but still fully functional. My editor formats stuff on save as well, it did not catch it and running goimports locally also does not correct it. I think it might be the explicit import name that makes it wonky. Either way, fixed now.

Reply:
This is not fixed now, sorry. As we are in an external project (fyne-x) the fyne.io imports come before github.com due to alphabetical ordering of external imports

Reply:
Good point. Thinking about this again, there probably should not be a newline between them, as we are not in the same repository.